.gitignore

```
@@ -1,4 +1,10 @@
 .idea/
-test-out/
-/testconf/
+/authaus.out
 /log/
+/testconf/
+test-out/
+/out
+/test.log
+/test-2025-08-07T06-17-16.613.log
+/test-2025-08-07T06-18-32.789.log
```

We should never have any `*.log` files in this repo. Please rather change this to
```suggestion
*.log
```